win32ss/user/ntuser/sysparams.c

```diff
@@ -301,7 +301,7 @@ SpiUpdatePerUserSystemParameters(VOID)
     gspv.bBeep = TRUE;
     gspv.uiFocusBorderWidth = 1;
     gspv.uiFocusBorderHeight = 1;
-    gspv.bMenuDropAlign = 1;
+    gspv.bMenuDropAlign = 0;
```

Comment:
Yeah I don't think we use this value at all other than in the code that reads / writes it.

Reply:
https://github.com/reactos/reactos/blob/master/win32ss/user/ntuser/sysparams.c#L897